types-bundle/package.json

```diff
@@ -46,9 +46,11 @@
   ],
   "author": "Moonsong Labs",
   "license": "GPL-3.0-only",
+  "homepage": "https://github.com/moonbeam-foundation/moonbeam/tree/master/types-bundle#readme",
```

Comment:
Shouldn't it be `https://github.com/moonbeam-foundation/moonbeam/tree/master/moonbeam-types-bundle#readme` ?

Reply:
Ah yeah, right